octopus/protocol/basic.py

```diff
@@ -10,7 +10,7 @@
 import logging
 
 # Package Imports
-from octopus.queue import AsyncQueue, AsyncQueueRetry
+from octopus.octopus_queue import AsyncQueue, AsyncQueueRetry
```

Comment:
I'm not sure I understand this - the module is octopus.queue.

Reply:
Fixed in https://github.com/richardingham/octopus/pull/9/commits/5bd3b5ab4e2ea9c7ca56064bc239136136fd72ea

I had renamed `queue.py` into `octopus_queue.py`, iirc because of some namespace collision. However that does not seem to be an issue anymore so back to old naming now